Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBar.xaml

```diff
@@ -0,0 +1,290 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls">
+
+    <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="ms-appx:///Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBarItem.xaml" />
+    </ResourceDictionary.MergedDictionaries>
+
+    <DataTemplate x:Key="NormalTabTemplate">
+        <NavigationViewItem Content="{Binding Header}" />
+    </DataTemplate>
+    <DataTemplate x:Key="ContextualTabTemplate">
+        <NavigationViewItem Content="{Binding Header}">
+            <NavigationViewItem.Resources>
+                <SolidColorBrush x:Key="TopNavigationViewItemForeground" Color="{ThemeResource SystemAccentColor}" />
+                <SolidColorBrush x:Key="TopNavigationViewItemForegroundSelected" Color="{ThemeResource SystemAccentColor}" />
+                <SolidColorBrush x:Key="TopNavigationViewItemForegroundPointerOver" Color="{ThemeResource SystemAccentColorLight2}" />
+                <SolidColorBrush x:Key="TopNavigationViewItemForegroundPressed" Color="{ThemeResource SystemAccentColorLight2}" />
+            </NavigationViewItem.Resources>
+        </NavigationViewItem>
+    </DataTemplate>
+    <controls:TabbedCommandBarItemTemplateSelector x:Key="DefaultTabbedCommandBarItemTemplateSelector"
+                                                   Contextual="{StaticResource ContextualTabTemplate}"
+                                                   Normal="{StaticResource NormalTabTemplate}" />
+
+    <StaticResource x:Key="TabbedCommandBarContentBackground" ResourceKey="SystemControlChromeLowAcrylicWindowBrush" />
+    <Style TargetType="controls:TabbedCommandBar">
+        <Setter Property="MenuItemTemplateSelector" Value="{StaticResource DefaultTabbedCommandBarItemTemplateSelector}" />
+        <Setter Property="PaneDisplayMode" Value="Top" />
+        <Setter Property="PaneToggleButtonStyle" Value="{StaticResource PaneToggleButtonStyle}" />
+        <Setter Property="IsTabStop" Value="False" />
+        <Setter Property="CompactPaneLength" Value="{ThemeResource NavigationViewCompactPaneLength}" />
+        <Setter Property="Background" Value="{ThemeResource SystemControlChromeMediumLowAcrylicWindowMediumBrush}" />
```

Comment:
I know this if fancy for apps which want to extend this into the titlebar, but we should default to the standard use-case of it fitting in with dropping into a blank app and use the existing resources like TabView or NavigationView use currently.

Developers can override this if they need to.

Reply:
The latest commit changes the default style to be the chrome (non-acrylic) equivalent of the previous defaults. That should solve this (and also not look as weird on the sample page). I supplied the acrylic version as a style that's based on the default style.